test/helpers/filterBuffer.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"bytes"
+	"strings"
+)
+
+// FilterBuffer structs that extends buffer methods
+type FilterBuffer struct {
+	*bytes.Buffer
+}
+
+// ByLines returns buf string plit by the newline characters
+func (buf *FilterBuffer) ByLines() []string {
```

Comment:
I'm not sure if it's worth it, or if it will break anything. Should we reset the buffer so we can call `ByLines` more than once?

Reply:
nope! I am wrong. https://play.golang.org/p/AdXIDrRzV4D ignore me